Individuals.cpp

```diff
@@ -58,4 +58,51 @@ void Individuals::addIndividual(Individual * ind)
 	ind->setNumericID( (unsigned int) num_samples++ );
 }
 
+void Individuals::loadOldIndividuals(string f) {
+	ifstream s_old(f.c_str());
+	if (!s_old) {
+		cerr << "WARNING: List of old individuals \"" << f << "\" could not be loaded" << endl;
+		return;
+	}
+
+	string fam_id, ind_id;
+
+	while(!s_old.eof()) {
+		s_old >> fam_id >> ind_id;
+                //cout << fam_id << ind_id << endl;
+                old_samples.insert(fam_id + " " + ind_id);
+	}
+}
+
+
+void Individuals::loadNewIndividuals(string f) {
+	ifstream s_new(f.c_str());
+	if (!s_new) {
+		cerr << "WARNING: List of new individuals \"" << f << "\" could not be loaded" << endl;
+		return;
+	}
+
+	string fam_id, ind_id;
+
+	while(!s_new.eof()) {
+		s_new >> fam_id >> ind_id;
+		new_samples.insert(fam_id + " " + ind_id);
+	}
+}
```

Comment:
Exposing my C++ ignorance here, but this seems like it should share implementation with loadOldIndividuals just above? Unless C++ makes reusing functions hard